packages/likec4/README.md

```diff
@@ -102,6 +102,10 @@ likec4 preview -o ./dist
 
 For example, this command can be used on CI, to compare diagrams with ones from the previous/main build.
 
+> **Tip:**\
+> The website root is strictly bound to the given base path (`/` by default).
+> If you need a relocatable bundle you may use `--base "./"`.
```

Comment:
I think makes sense to mention in docs
https://github.com/likec4/likec4/blob/7b6921b1f6d4b4db87fea972c6524986e5ac1f62/apps/docs/src/content/docs/tooling/cli.mdx?plain=1#L98-L101

Reply:
...just updated `cli.mdx`; More or less copy & paste.